frontend/src/fixtures/commonsFixtures.js

```diff
@@ -4,37 +4,34 @@ const commonsFixtures = {
             "id": 5,
             "name": "Seths Common",
             "day": 5,
-            "startDate": "2022-03-05T15:50:10",
-            "endDate": "2023-03-05T15:50:10",
+            "startDate": "6/10/2021",
```

Comment:
Per Xiyue, these should be ISO dates; that's what's returned by the backend.

So let's make the fixtures match